src/modules/fw_att_control/fw_att_control_params.c

```diff
@@ -616,10 +616,12 @@ PARAM_DEFINE_INT32(FW_BAT_SCALE_EN, 0);
  *
  * @min 45
  * @max 720
- * @unit deg
+ * @decimal 1
```

Comment:
Does it make sense to set tenth of degrees per second? I would leave it float but not show decimals.
```suggestion
 * @decimal 0
```

Reply:
I aligned with the MC rate controller params, so e.g. MC_ACRO_R_MAX

Reply:
Well, there it's IMHO also too fine-grained then. I wouldn't know when to use 0.1 degree/s. I can suggest that separately.